hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -302,6 +310,31 @@ public void frameworkMessage(ExecutorDriver driver, byte[] msg) {
     log.info(String.format("Received framework message: %s", messageStr));
   }
 
+  private String getNodeId() {
+    String id = null;
```

Comment:
@elingg 
Is it possible to access ZK state on Executor's side? I didn't saw such access anywhere in the code. And this seems to be a design decision, isn't it?